src/scripts.js

```diff
@@ -444,10 +442,10 @@ const recipeToCookRemovalConfirmation = (recipe) => {
 // cookListActionButton
 const cookListAddRemoveHandler = () => {
   let selectedRecipe = findRecipeWithID(parseInt(cookCardActionButton.id))
-  if (cookListAddRemoveButton.classList.value === "cook-list-action add") {
+  if (cookListAddRemoveButton.className.includes('cook-list-action')) {
     let missingIngredients = currentUser.returnMissingIngredientsFor(selectedRecipe);
```

Comment:
The word "add" is necessary in this classList, it distinguishes the two pathways here. Otherwise the remove button will not work. Will adding this back in affect your functionality in any way?